governance.md

```diff
@@ -264,47 +264,162 @@ interactions with NumFOCUS.
+If a DPL position is not filled, the responsibility devolves back to
+the PL.
 
-An individual may hold more than one DLD simultaneously.
+At each 1yr term a DPL has the option to continue for another year or
+step down.
 
-### Release Manager
+The SC can remove a DPL using the same process as ejecting a Steering
+Council Member.
 
+The SC can create a new DPL position or eliminate an unfilled DPL
+position by majority vote.
 
-### Narrative Documentation Lead
 
-### API Documentation Lead
+### Release Manager(s)
 
-### Communication and Community Lead
+The Release Manager (RM) is appointed for a minor version (A.B.x)
+release series of Matplotlib.  They are responsible for the full
+release life cycle of all minor releases in the series including:
 
-### API consistency Lead
+- ensuring the whats new, API changes, and release notes are up to date
+- the timing of the releases
+- what changes should or should not be backported from the master
+  branch
+- rebuilding and publishing the website
+- announcing the release
+- publishing sdist and wheels to pypi
+- notifying down-stream packagers of the release
+
+An individual can be the RM as more than one release series at the
+same time.
+
+
+### Narrative Documentation Manager
+
+Matplotlib has a tremendous amount of documentation that is narrative
+in form.  This includes our examples and tutorials that live inside the
+main source repository and longer tutorials that live in other repositories
+in the Matplotlib organization.
+
+The Narrative Documentation Matplotlib (NDM) is responsible for
+shepherding all of this content including the scope, level, tone, and
+voice.
+
+### Reference Documentation Manager
+
+Matplotlib also has a tremendous amount of reference documentation
+embedded in doc strings.  This documentation needs to be complete and
+accurate as our users rely on it as the last authority of what a given
+method will do (short of reading the source).
+
+The Reference Documentation Manager (RDM) is responsible for ensuring that
+the docstrings are:
+
+- correctly formatted and render as intended
+- technically correct
+- complete
+
+### Community Manager
+
+The true strength of Matplotlib and why it has had such longevity as a
+project is the community of people around the code.  That community
+needs to be maintained.  The Community Manager (CM) is the is a
+catch-all for several very diverse tasks and this role may be split
+in the future and may want to enlist further assistants.
+
+**Communications**
+
+We need to maintain communication channels with the wider world.  To this end
+the CM is responsible for maintaining:
+
+- Matplotlib's social media presence
+- The Matplotlib blog
+- Recruiting people to give talks on Matplotlib at conferences / meetups
+- Maintaining a "matplotlib update" slide deck
+- the [Ann] mailing list
+
+This communication is primarily broadcast out and intended to engage
```

Comment:
and developers of 3rd party libraries. Also I just don't like the language of end users, user/developer split.

Reply:
I am not a fan either, but was having trouble finding other words for it.

Reply:
community